.github/workflows/ports_esp32.yml

```diff
@@ -34,6 +34,7 @@ jobs:
       run: source tools/ci.sh && echo "IDF_VER=$IDF_VER" | tee "$GITHUB_OUTPUT"
 
     - name: Cached ESP-IDF install
+      if: 0  # Temporary: skip cache for PR verification
```

Comment:
Does this need to be permanently disabled to get CI to work, or can this line be removed now?